templates/projects/list.tmpl

```diff
@@ -38,7 +38,11 @@
 		<div class="milestone list">
```

Comment:
Yes, it's a blatant copy as seen here. Possible to fix these to `projects`? CSS may need adjustment.